src/libraries/Common/src/Interop/Unix/System.Native/Interop.IOVector.cs

```diff
@@ -9,8 +9,10 @@ internal static partial class Sys
     {
         internal unsafe struct IOVector
```

Comment:
As an alternative to disabling the `CS0649` warning, you can set the layout type to sequential for the same effect. This appears to be more common in the ` src/libraries/Common/src/Interop` folder.

```suggestion
        [StructLayout(LayoutKind.Sequential)]
        internal unsafe struct IOVector
```

You will also need to add an using:

```c#
using System.Runtime.InteropServices;
```

Reply:
updated.